stdlib/Random/src/RNGs.jl

```diff
@@ -336,7 +336,7 @@ typeof_rng(::_GLOBAL_RNG) = TaskLocalRNG
 Return the default global random number generator (RNG).
 
 !!! note
-    What the default RNG is is an implementation detail.  Across different versions of
+    The default RNG is an implementation detail. Across different versions of
```

Comment:
I find this wording unclear: is the existence of the default RNG an implementation detail? no, it's documented to exist and be returned by `default_rng()`. What is an implementation detail is the implementation of this default RNG, i.e. its type.  See #52385 for a rewording.